launcher/src/components/base/BaseLayout.vue

```diff
@@ -190,6 +190,11 @@ const checkSettings = async () => {
         await ControlService.idleTimerCheck(false);
       }
     }
+    if (typeof savedConfig.logBackups.value !== "undefined") {
```

Comment:
It does not really make sense to store this detail on the server when it's affecting files on the local machine so independent of any server